broker-core/src/main/java/io/zeebe/broker/system/management/topics/FetchCreatedTopicsRequestHandlerService.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Zeebe Broker Core
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.zeebe.broker.system.management.topics;
+
+import io.zeebe.broker.clustering.base.partitions.Partition;
+import io.zeebe.broker.clustering.orchestration.state.KnownTopics;
+import io.zeebe.broker.system.management.LeaderManagementRequestHandler;
+import io.zeebe.servicecontainer.Injector;
+import io.zeebe.servicecontainer.Service;
+import io.zeebe.servicecontainer.ServiceGroupReference;
+
+public class FetchCreatedTopicsRequestHandlerService
```

Comment:
This looks unnecessarily complex. Why have an extra service and not just merge it with `LeaderManagementRequestHandler`?